pyproject.toml

```diff
@@ -107,6 +107,18 @@ warn_no_return = true
 warn_unused_configs = true
 warn_unused_ignores = false
 
+[[tool.mypy.overrides]]
+module = ["onnxscript.tools.benchmark.benchmark_helpers"]
+disable_error_code = 'name-defined'  # raised due to annotation as string
+
+[[tool.mypy.overrides]]
+module = ["onnxscript.tools.benchmark.export_model_batch"]
+disable_error_code = 'import-untyped'
+
+[[tool.mypy.overrides]]
+module = ["onnxscript.tools.transformers_models"]
+disable_error_code = 'import-untyped'  # raised due to annotation as string
+
```

Comment:
```suggestion
```

I would add in the files to keep this config file stable. Otherwise this list will grow very quickly

Reply:
What is the syntax to disable inline?

Reply:
## To fix these (preferred)

`import-untyped` can be fixed with adding the package line to https://github.com/microsoft/onnxscript/blob/1efd8e665159bb3fe170628a923db435e165942f/pyproject.toml#L33-L34

```
some_package = ["py.typed"]
```

`name-defined` can be fixed with importing under `if typing.TYPE_CHECKING:` so the module is only imported for static checkers

## To disable inline

```
# type: ignore[error_code]
```

## To disable for the whole file

```
# mypy: disable-error-code=override
```

E.g. https://github.com/microsoft/onnxscript/blob/1efd8e665159bb3fe170628a923db435e165942f/onnxscript/onnx_opset/_impl/opset10.py#L9

Reply:
removed but I still don't know what to do when ruff and mypy complain on the same line.